packages/paste-website/src/pages/components/chat-log/index.mdx

```diff
@@ -451,15 +449,17 @@ ChatAttachmentLink inherits the same props as [Anchor](/components/anchor) as we
 ### Figma
 
 <Callout variant="neutral" marginY="space70">
-  <CalloutHeading as="h4">Chat Log components are not in the Paste Component Library yet</CalloutHeading>
+  <CalloutHeading as="span">Figma usage guidelines coming soon</CalloutHeading>
   <CalloutText>
-    To find the Stage 1 components, see{' '}
-    <Anchor href="https://www.figma.com/file/cSdKlqoTbxoAUmnrYkfFWl/Conversations-UI-Kit?node-id=0%3A1" showExternal>
-      the Conversations UI Kit Figma file
+    You can find the Chat Log components in{' '}
+    <Anchor
+      href="https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=16157%3A123797"
+      showExternal
+    >
+      the Paste Components library
     </Anchor>
     .
-  </CalloutText>
-</Callout>
+  </Callout>
```

Comment:
question: is this diff being weird, or is the closing `</CalloutText>` missing?

Reply:
No it was missing lol